lib/OpenQA/Script/CloneJob.pm

```diff
@@ -141,6 +141,7 @@ sub clone_job_download_assets ($jobid, $job, $url_handler, $options) {
 
             print STDERR "downloading\n$from\nto\n$dst\n";
             my $r = $ua->mirror($from, $dst);
+            $r = $ua->mirror($r->header('Location'), $dst) if ($r->code == 308);
```

Comment:
What if `Location` is `undef`? What if there's yet another redirection? We should also not follow a location from https to http.

Reply:
But generally a good idea of course.

Reply:
@Martchus FYI in my case it was the redirect from http to https on o3.

Reply:
Maybe add a comment that this is a temporary fix until LWP 6.48 is available on all our supported platforms?

Reply:
@kalikiana I suppose this is expected you add before merge, right @okurz ?

Reply:
yes, or anyone else who can add a commit to this PR

Reply:
Apparently we can't achieve with a reasonable effort to update LWP in SLE/Leap hence we need to keep this workaround in place until eventually we use only more recent versions in newer OS versions


```suggestion
            # Workaround for behaviour LWP < 6.48, see https://github.com/libwww-perl/libwww-perl/pull/349
            $r = $ua->mirror($r->header('Location'), $dst) if $r->code == 308;
```

Reply:
@pevik Maybe it's best if you take over from here? I don't even remember what issue this was addressing.